script.py

```diff
@@ -1,73 +1,67 @@
-import time 
-from selenium import webdriver 
+import time
+from selenium import webdriver
 from selenium.webdriver.support.select import Select
 
-driver = webdriver.Chrome()
+driver = webdriver.Chrome('D:/webdriver/chromedriver.exe')
 url = "https://gu.icloudems.com/corecampus/index.php"
-  
-# Opening the website 
-driver.get(url) 
+
+# Opening the website
+driver.get(url)
 driver.maximize_window()
 
 time.sleep(2)
 username = driver.find_element_by_xpath('//*[@id="useriid"]')
 password = driver.find_element_by_xpath('//*[@id="actlpass"]')
 
-
-#Change user name and passwords with your icloud username and password
-username.send_keys('username')  ##################################### Change This Text ############################
-password.send_keys('password')  ##################################### Change This Text ############################
+# Change user name and passwords with your icloud username and password
+username.send_keys('17SCSE112003')  ##################################### Change This Text ############################
+password.send_keys('Gourav8211@')  ##################################### Change This Text ############################
```

Comment:
You should remove this username and password and replace them with a placeholder.